step2_PAT.py

```diff
@@ -37,6 +37,11 @@
     ),
     secondaryFileNames = cms.untracked.vstring()
 )
+"""
+process.source = cms.Source("PoolSource",
+    fileNames = cms.untracked.vstring('file:/eos/uscms/store/user/wmccorma/009E7EE3-3781-5048-A2F2-0E6139B13D46.root'),
```

Comment:
not for general usage?

Reply:
ah right, since jobs weren't working on hammer, I switched to lpc, where the remote files don't work.  Fixed this now

Reply:
there is no reason the remote files should not work at lpc.

Reply:
Huh ok, I think it's working now... maybe there was a transient issue retrieving files from Purdue on Monday?  This is what I was getting from the job previously:
```
Exception Message:
Failed to open the file 'root://cmsxrootd.fnal.gov//store/mc/RunIISummer20UL17RECO/TTJets_TuneCP5_13TeV-amcatnloFXFX-pythia8/AODSIM/106X_mc2017_realistic_v6-v2/230000/009E7EE3-3781-5048-A2F2-0E6139B13D46.root'
   Additional Info:
      [a] Calling RootInputFileSequence::initTheFile(): fail to open the file with name root://cmsxrootd-site.fnal.gov//store/mc/RunIISummer20UL17RECO/TTJets_TuneCP5_13TeV-amcatnloFXFX-pythia8/AODSIM/106X_mc2017_realistic_v6-v2/230000/009E7EE3-3781-5048-A2F2-0E6139B13D46.root
      [b] Input file root://cmsxrootd.fnal.gov//store/mc/RunIISummer20UL17RECO/TTJets_TuneCP5_13TeV-amcatnloFXFX-pythia8/AODSIM/106X_mc2017_realistic_v6-v2/230000/009E7EE3-3781-5048-A2F2-0E6139B13D46.root could not be opened.
      [c] XrdCl::File::Open(name='root://cmsxrootd.fnal.gov//store/mc/RunIISummer20UL17RECO/TTJets_TuneCP5_13TeV-amcatnloFXFX-pythia8/AODSIM/106X_mc2017_realistic_v6-v2/230000/009E7EE3-3781-5048-A2F2-0E6139B13D46.root', flags=0x10, permissions=0660) => error '[ERROR] Operation expired' (errno=0, code=206). No additional data servers were found.
      [d] Last URL tried: root://cms-xrd-global.cern.ch:1094//store/mc/RunIISummer20UL17RECO/TTJets_TuneCP5_13TeV-amcatnloFXFX-pythia8/AODSIM/106X_mc2017_realistic_v6-v2/230000/009E7EE3-3781-5048-A2F2-0E6139B13D46.root?tried=+1213cmsxrootd2.fnal.gov1213xrootd.unl.edu,af-a00.cms.rcac.purdue.edu&xrdcl.requuid=e27fc2a5-f3df-43e8-8a2b-05ff8b52809c
      [e] Problematic data server: cms-xrd-global.cern.ch:1094
      [f] Disabled source: cms-xrd-global.cern.ch:1094
```

Reply:
I reran the same command that just worked and of course now it seems to be getting stuck again.  It's been sitting with the following displayed for the past 45 minutes or so...
`[2023-11-15 12:24:01.140600 -0600][Error  ][PostMaster        ][16940] [af-a00.cms.rcac.purdue.edu:1098] Forcing error on disconnect: [ERROR] Operation interrupted.
[2023-11-15 12:38:55.008711 -0600][Error  ][PostMaster        ][16940] [cmsxrootd-site.fnal.gov:1094] Forcing error on disconnect: [ERROR] Operation interrupted.
[2023-11-15 12:39:01.767240 -0600][Error  ][PostMaster        ][16940] [af-a00.cms.rcac.purdue.edu:1094] Forcing error on disconnect: [ERROR] Operation interrupted.
[2023-11-15 12:39:01.806260 -0600][Error  ][PostMaster        ][16940] [cmsxrootd.fnal.gov:1094] Forcing error on disconnect: [ERROR] Operation interrupted.`

Reply:
If you want to ensure reading the files from T2_US_Purdue (rather than going through the redirector, which could send you to T2_IT_Pisa, which then might time out), based on `/cvmfs/cms.cern.ch/SITECONF/T2_US_Purdue/storage.json`, the correct prefix is: `root://eos.cms.rcac.purdue.edu/`. This can be prepended to the LFN for each file in the list.

Reply:
That seems to work consistently, so I added it in